packages/hs-react-ui/src/components/Button/Button.tsx

```diff
@@ -60,12 +67,17 @@ export const getFontColorFromVariant = (variant: string, color: string) => {
  * Get the appropriate background color for the button based on the variant of button
  * @param {string} variant - The variant of button
  * @param {string} color - The color prop passed into the button
+ * @param {string} transparentColor - The color to use for a transparent background
  */
-export const getBackgroundColorFromVariant = (variant: string, color: string) => {
+export const getBackgroundColorFromVariant = (
+  variant: string,
+  color: string,
+  transparentColor: string,
+) => {
   switch (variant) {
     case ButtonVariants.text:
     case ButtonVariants.outline:
-      return colors.transparent;
+      return transparentColor;
```

Comment:
if it's optional, should it be more like
```suggestion
      return transparentColor || 'transparent';
```

Couldn't do the same for color as we don't have a sensible default we can recommend, and transparent is brand neutral so prob safe. Should probably make color and variant required in the jsdoc/typing

Reply:
Change made